api/DTOs/SupplierDTO.cs

```diff
@@ -0,0 +1,28 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace DTO.Supplier;
+
+[ApiExplorerSettings(IgnoreApi = true)]
+public class SupplierRequest : BaseDTO
+{
+    public string? Code { get; set; }
+    public string? Name { get; set; }
+    public string? Reference { get; set; }
+    public Guid? Contact_id { get; set; }
+    public ContactDTO? Contact { get; set; }
+    public Guid? Address_id { get; set; }
+    public AddressDTO? Address { get; set; }
+}
+
+[ApiExplorerSettings(IgnoreApi = true)]
+public class SupplierResponse : BaseDTO
+{
+    public Guid Id {get; set;}
+    public string? Code { get; set; }
+    public string? Name { get; set; }
+    public string? Reference { get; set; }
+    public ContactDTO? Contact { get; set; }
+    public AddressDTO? Address { get; set; }
+    public string? Created_at { get; set; }
+    public string? Updated_at{get;set;}
```

Comment:
```suggestion
    public string? CreatedAt{ get; set; }
    public string? UpdatedAt{get;set;}
```

Graag PascalCase aanhouden